tests/test_update.py

```diff
@@ -108,8 +123,8 @@
     {"message": message},
     {"edited_message": message},
     {"callback_query": CallbackQuery(1, User(1, "", False), "chat", message=message)},
-    {"channel_post": message},
-    {"edited_channel_post": message},
+    {"channel_post": channel_post},
+    {"edited_channel_post": channel_post},
```

Comment:
Had to create separate `message` objects here for `channel_post`s, since they always have `sender_chat` and should not be deleted as will happen to regular messages below